### ISSUE TITLE
The css() function doesn't output the hashed stylesheet name when using postcss

### ISSUE THREAD

**Issue by ay13** [carrot/roots-css-pipeline#22]
In `app.production.coffee` I have:
`css_pipeline(files: 'assets/css/*.scss', postcss: true, out: 'css/build.css', minify: true, hash: true)`

After `roots compile --env production` my `/public/css` directory has the hashed versions but the `css()` function in jade outputs `<link rel='stylesheet' href='/css/build.css' />` (without the hashes).

Because of this the css never loads.

**Comment by vipul261** [comment]
I am also having the same issue. In my case I dont have the postcss option

**Comment by jescalan** [comment]
Yeah, `postcss` isn't actually an option, so slightly confused about that.

The hashing logic is [here](https://github.com/carrot/roots-css-pipeline/blob/master/lib/index.coffee#L93), and the `css()` local function definition is [here](https://github.com/carrot/roots-css-pipeline/blob/master/lib/index.coffee#L45) -- the fix should be separating the hashing logic into a function and adding it to the css local. Anyone want to take a stab at a PR fix? This should be a nice easy one to start on 😁

**Comment by vipul261** [comment]
@jescalan I can give it a shot. Though I'm not familiar with coffeescript and this would be my first contribution so far.

**Comment by jescalan** [comment]
@vipul261 no better time to start than now! 🎉 

Let me know if you need any help along the way
